web_client/src/lib/agents/Pacman.svelte

```diff
@@ -82,7 +82,7 @@
 
 {#if showPacman}
   <div
-    class='pacman {clip ? 'clip' : ''} eating'
+    class={`pacman ${clip ? 'clip' : ''} eating`}
```

Comment:
I think this change is unnecessary, as Svelte's compiler already recognizes the braces as a string interpolation. In the web client, the behavior of both would be the same.